automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/billing/BillingManagementPage.scala

```diff
@@ -54,145 +50,73 @@ class BillingManagementPage(implicit webDriver: WebDriver) extends Authenticated
     * @return a status of "success", "failure", or "unknown"
     */
   def waitForCreateCompleted(projectName: String): String = {
-    filter(projectName)
+    billingProjectTable.filter(projectName)
     retry(10.seconds, 5.minutes)({
-          ui.readCreationStatusForProject(projectName).filterNot(_ equals "running")
+          readCreationStatusForProject(projectName).filterNot(_ equals "running")
         }) match {
       case None => throw new Exception("Billing project creation did not complete")
       case Some(s) => s
     }
   }
 
+  private def readCreationStatusForProject(projectName: String): Option[String] = {
+    for {
+      e <- find(xpath(s"//div[@data-test-id='$projectName-row']//span[@data-test-id='status-icon']"))
+      v <- e.attribute("data-test-value")
+    } yield v
+  }
+
 
-  def openBillingProject(projectName: String) = {
-    filter(projectName)
-    ui.openBillingProject(projectName)
+  def openBillingProject(projectName: String): Unit = {
+    billingProjectTable.filter(projectName)
+    val billingProjectLink = testId(projectName + "-link")
+    click on (await enabled billingProjectLink)
   }
 
 
-  def addUserToBillingProject(userEmail: String, role: String) = {
-    val modal = ui.openAddUserDialog()
+  def addUserToBillingProject(userEmail: String, role: String): Unit = {
+    addUserButton.doClick()
+    val modal = await ready new AddUserToBillingProjectModal
```

Comment:
i'm a little confused as to why we don't use open here like we do for page objects

Reply:
Modals aren't Pages and don't have the `open` method. We could change that.